hydra-node/src/Hydra/Ledger.hs

```diff
@@ -52,6 +54,12 @@ instance ToCBOR Party where
 newtype Signed a = UnsafeSigned (SigDSIGN MockDSIGN)
   deriving (Eq, Show)
 
+instance Arbitrary (Signed a) where
+  arbitrary = do
```

Comment:
Should: add a comment that this arbitrary instance is producing always invalid signatures.